lkmpg.tex

```diff
@@ -835,6 +835,9 @@ \subsection{The file\_operations Structure}
 
 An instance of \cpp|struct file_operations| containing pointers to functions that are used to implement \cpp|read|, \cpp|write|, \cpp|open|, \ldots{} system calls is commonly named \cpp|fops|.
 
+Since Linux v3.14, the read, write and seek operations are guaranteed for thread-safe by letting the file position update being mutual exclusion.
```

Comment:
Avoid using `letting` since we need the subject and/or someone to perform the action.